examples/tutorials/streamwriter_advanced.py

```diff
@@ -0,0 +1,326 @@
+#
+#    To install torchaudio nightly build, please refer to
+#    https://pytorch.org/get-started/locally/ .
+#
+#
+#    There are multiple ways to install FFmpeg libraries.
+#    If you are using Anaconda Python distribution,
+#    ``conda install 'ffmpeg<4.4'`` will install the required FFmpeg libraries,
+#    however, this distribution does not have SDL plugin, so it cannot play
+#    video.
+#
+
+######################################################################
+#
+# .. warning::
+#
+#    TorchAudio dynamically loads compatible FFmpeg libraries
+#    installed on the system.
+#    The types of supported formats (media format, encoder, encoder
+#    options etc) depend on the libraries.
+#
+#    To check the available devices, muxers and encoders, you can use the
+#    following command
+#
+#    .. code-block:: console
+#
+#       ffmpeg -muxers
+#       ffmpeg -encoders
+#       ffmpeg -devices
+#       ffmpeg -protocols
+
+######################################################################
+#
+# Preparation
+# -----------
+
+import torch
+import torchaudio
+
+print(torch.__version__)
+print(torchaudio.__version__)
+
+from torchaudio.io import StreamWriter
+
+######################################################################
+#
+
+from torchaudio.utils import download_asset
+
+AUDIO_PATH = download_asset("tutorial-assets/Lab41-SRI-VOiCES-src-sp0307-ch127535-sg0042.wav")
+VIDEO_PATH = download_asset("tutorial-assets/stream-api/NASAs_Most_Scientifically_Complex_Space_Observatory_Requires_Precision-MP4_small.mp4")
+
+######################################################################
+#
+# Device Availability
+# -------------------
+#
+# ``StreamWriter`` takes advantage of FFmpeg's IO abstraction and
+# write the data to media devices such as speakers and GUI.
+#
+# To write to devices, provide ``format`` option to the constructor
+# of ``StreamWriter``.
+#
+# Different OS has different device option and their availabilities
+# depend on the actual installation of FFmpeg.
+#
+# To check which device is available, you can use `ffmpeg -devices`
+# command.
+#
+# "audiotoolbox" (speaker) and "sdl" (video GUI)
+# are available.
+#
+# .. code-block:: console
+#
+#    $ ffmpeg -devices
+#    ...
+#    Devices:
+#     D. = Demuxing supported
+#     .E = Muxing supported
+#     --
+#      E audiotoolbox    AudioToolbox output device
+#     D  avfoundation    AVFoundation input device
+#     D  lavfi           Libavfilter virtual input device
+#      E opengl          OpenGL output
+#      E sdl,sdl2        SDL2 output device
+#
+# For details about what devices are available on which OS, please check
+# the official FFmpeg documentation. https://ffmpeg.org/ffmpeg-devices.html
+#
+
+######################################################################
+#
+# Playing audio
+# -------------
+#
+# By providing `format="audiotoolbox"` option, the StreamWriter writes
```

Comment:
not sure if there is a way to force it to be a left quote character instead of a right quote
![Uploading Screen Shot 2022-10-02 at 10.06.43 AM.png…]()